client/templates/home/home.js

```diff
@@ -1,3 +1,4 @@
+Meteor.subscribe("leaderboardSortedTeams");
```

Comment:
If we are going to remove the buttons from home screen then the team subscription wouldn't make sense

also, with the aggregated teams you can mimic the functionality of `TeamUtils.isInTeam(userId)` , `getTeam(userId)` and `isScrum(userId, teamId)` functions instead of using those if needed, the router subscription will manage to make `waitOn` run normally